### ISSUE TITLE
Queue problem

### ISSUE THREAD

**Issue by immortaljam** [OuterTune/OuterTune#77]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/dd3boh/OuterTune/actions).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

go to playlist
add song to queue

### Expected behavior

when adding a song to queue, it should go to the bottom queue

### Actual behavior

instead of the song getting added to the bottom of the queue it plays right after the current song finishes

### Screenshots/Screen recordings

![Screenshot_20241026-105550_OuterTune](https://github.com/user-attachments/assets/7fefaa4e-8a60-45da-918c-8fb2f1169bb6)
![Screenshot_20241026-105554_OuterTune](https://github.com/user-attachments/assets/49b5003d-3a31-404d-a579-defb43b77004)
![Screenshot_20241026-105600_OuterTune](https://github.com/user-attachments/assets/3434d368-94e2-4c68-ae9c-fc9314bbc695)

### Logs

idk how to do this

### OuterTune version

0.6.2(23) universal

### Android version

12

### Additional information

I love this app, tysm for all ur help

**Comment by AKoreanDumpling** [comment]
That swipe action is actually "play next", which adds the song directly below the currently playing one.

**Comment by immortaljam** [comment]
> That swipe action is actually "play next", which adds the song directly below the currently playing one.

yea I realized
but an option to choose what swiping right should do would be a nice feature

**Comment by 7heMech** [comment]
> an option to choose what swiping right should do would be a nice feature 

Yeah, that would be for the best, but probably hard to implement?
